src/pages/gen2/build-a-backend/data/subscribe-data/index.mdx

```diff
@@ -116,7 +116,7 @@ If you want to get all subscription events, don't specify any `filter` parameter
 **Limitations:**
 
 - Specifying an empty object `{}` as a filter is **not** recommended. Using `{}` as a filter might cause inconsistent behavior based on your data model's authorization rules.
-- If you are using dynamic group authorization and you authorize based on a single group per record, subscriptions are only supported if the user is part of five or fewer user groups.
+- If you're using dynamic group authorization and you authorize based on a single group per record, subscriptions are only supported if the user is part of five or fewer user groups.
 - Additionally, if you authorize by using an array of groups (`groups: [String]`),
   - subscriptions are only supported if the user is part of 20 or fewer groups
   - you can only authorize 20 or fewer user groups per record
```

Comment:
@renebrandel The links in the next steps section refer to gen1 docs. I was going to update these to gen2 docs but realized that they've been descoped for gen2. Can you provide other gen2 links that would work here? Thanks!